src/client/components/description/head/Head.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import searchIcon from '@root/client/images/search.png';
-import styles from './Head.module.css';
+import './Head.module.css';
 
 const Head = (props) => {
```

Comment:
you don't use here props so no need to pass it here

Reply:
Repaired